bdb-ip.md

```diff
@@ -0,0 +1,1000 @@
+I want to retrieve all Ed25519 outputs from multiple transactions for given public keys
+so that I can create a follow-up TRANSFER transaction.
+
+
+##### Scenario 1.1: Query for single either spent or unspent Ed25519 output
+
+Given a valid transaction with a single either spent or unspent Ed25519 output exists
+When a BigchainDB user is querying BigchainDB for an either spent or unspent output with a specific public key
+Then BigchainDB returns all required fractals of information to generate a follow-up TRANSFER transaction for the existing either spent or unspent output.
+
+
+##### Scenario 1.2: Query for multiple either spent or unspent Ed25519 outputs with single distinct public key
+
+Given a valid transaction with multiple unspent Ed25519 outputs exists
+When a BigchainDB user is querying BigchainDB for an unspent output with a specific public key
+Then BigchainDB returns all required fractals of information to generate a follow-up TRANSFER transaction for the existing either spent or unspent output.
+
+
+##### Scenario 1.3: Query for multiple either spent or unspent Ed25519 outputs with multiple distinct public keys
+
+Given a valid transaction with multiple either spent or unspent Ed25519 outputs exists
+When a BigchainDB user is querying BigchainDB for either spent or unspent outputs with specific public keys
+Then BigchainDB returns all required fractals of information to generate a follow-up TRANSFER transaction for the existing either spent or unspent outputs.
+
+
+##### Scenario 1.4: Query for multiple either spent or unspent Ed25519 outputs with single distinct public keys
+
+Given a valid transaction with multiple either spent or unspent Ed25519 outputs exists
+When a BigchainDB user is querying BigchainDB for either spent or unspent outputs with a specific public key
+Then BigchainDB returns all required fractals of information to generate a follow-up TRANSFER transaction for the existing unspent output.
+
+
+##### Scenario 1.5: Query for multiple either spent or unspent Ed25519 outputs with multiple keys where some outputs are missing
+
+Given one or more valid existing transactions with either spent or unspent Ed25519 outputs
+When a BigchainDB user is querying BigchainDB for either spent or unspent outputs with specific public keys
+And some specific public keys do not match any existing either spent or unspent outputs in BigchainDB
+Then BigchainDB returns for all either spent or unspent outputs matching specific public keys all required fractals of information to generate a follow-up TRANSFER transaction
+And the BigchainDB user is not notified about potentially unmatched either spent or unspent outputs.
+
+
+#### Use case 2: Sha256Threshold outputs
+
+As a BigchainDB user
+I want to retrieve all Sha256Threshold outputs from multiple transactions for given public keys
+so that I can create a follow-up TRANSFER transaction.
+
+
+##### Scenario 2.1: Query for single Sha256Threshold output with single public key
```

Comment:
Does this mean a treshold condition with multiple ed25519 subconditions for the same public key?